wdl-lint/src/rules/mixed_indentation.rs

```diff
@@ -53,8 +54,8 @@ impl From<u8> for IndentationKind {
     }
 }
 
-/// Creates a "mixed indentation" diagnostic.
-fn mixed_indentation(command: Span, span: Span, kind: IndentationKind) -> Diagnostic {
```

Comment:
this should still be one function named `mixed_indentation`. It can take a fourth arg which is `warning: Bool` instead of writing a near duplicate function